.github/workflows/dist.yml

```diff
@@ -61,3 +61,52 @@ jobs:
           name: input-leap-rpms-${{ matrix.rpm_tag }}
           path: |
             rpms/**/*.rpm
+
+  flatpak:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: ./input-leap
+    steps:
+      # submodules require git to be installed
+      - name: Check out repository
+        uses: actions/checkout@v4
+        with:
+          path: input-leap
+          # submodules: recursive
+          set-safe-directory: ${{ github.workspace }}
+
+      - run: git config --global protocol.file.allow always
+
+      - name: Install flatpak and flatpak builder
+        run: sudo apt-get install flatpak flatpak-builder
+
+      - name: add flatpak remote
+        run: sudo flatpak remote-add --if-not-exists flathub https://flathub.org/repo/flathub.flatpakrepo
+
+      - name: Install flatpak dependencies
+        run: sudo flatpak install -y flathub org.kde.Sdk//5.15-23.08 org.kde.Platform//5.15-23.08
```

Comment:
Have you considered using the [`flatpak/flatpak-builder`](https://github.com/marketplace/actions/flatpak-builder) action instead of this? Should save some bandwidth

Reply:
Why does the action save bandwidth?

Reply:
I'll take a look though I'm not sure how it will handle installing custom SDKs unless they have an image for each one.

Reply:
OK this is def the right way to go. Let me change this.

Reply:
Fixed